Documentation/contributing.rst

```diff
@@ -1027,8 +1012,84 @@ requirements have been met:
   Certificate of Origin" for more details.
 
 
+CI / Jenkins
+------------
+
+The main CI infrastructure is maintained at https://jenkins.cilium.io/. 
+
+Jobs Overview
+~~~~~~~~~~~~~
+
+Cilium-PR-Ginkgo-Tests-Validated
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Runs validated Ginkgo tests which are confirmed to be stable and have been
+verified. These tests must always pass.
+
+The configuration for this job is contained within ``ginkgo.Jenkinsfile``.
+
+It first runs unit tests using docker-compose using a YAML located at
+``test/docker-compose.yaml``.
+
+The next steps happens in parallel:
+
+    - Runs the runtime e2e tests.
```

Comment:
I'd like to move away from 'runtime' e2e tests. Can you specify that it tests against the Docker runtime instead of just 'runtime'?